build_tools/cmake/test_riscv.sh

```diff
@@ -14,8 +14,13 @@ set -xeuo pipefail
 export PS4='[$(date -u "+%T %Z")] '
 
 ROOT_DIR="${ROOT_DIR:-$(git rev-parse --show-toplevel)}"
-IREE_IMPORT_TFLITE_BIN="${IREE_IMPORT_TFLITE_BIN:-iree-import-tflite}"
-LLVM_BIN_DIR="${LLVM_BIN_DIR}"
+
+RISCV_ARCH="${RISCV_ARCH:-rv64}"
+
+if [[ "${RISCV_ARCH}" == "rv64" ]]; then
+  IREE_IMPORT_TFLITE_BIN="${IREE_IMPORT_TFLITE_BIN:-iree-import-tflite}"
+  LLVM_BIN_DIR="${LLVM_BIN_DIR}"
+fi
```

Comment:
This if condition and the one at L65-82 will be removed once `iree-run-module-test` PR is landed.